rules-tests/PSR4/FileRelocationResolverTest.php

```diff
@@ -34,6 +34,9 @@ public function test(string $file, string $oldClass, string $newClass, string $e
             $newClass
         );
 
+        // we expect a path containing the dir-separator of the current platform
+        $expectedNewFileLocation = str_replace("/", DIRECTORY_SEPARATOR, $expectedNewFileLocation);
```

Comment:
I figured that the expected behaviour of `fileRelocationResolver` is, that it should return the path with a DIRECTORY_SEPARATOR, not always with `/` like the test asserted before